webv2/updateTableSchema/interleavedreview_test.go

```diff
@@ -0,0 +1,143 @@
+package updateTableSchema
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/bmizerany/assert"
+	"github.com/cloudspannerecosystem/harbourbridge/common/constants"
+	"github.com/cloudspannerecosystem/harbourbridge/internal"
+	"github.com/cloudspannerecosystem/harbourbridge/spanner/ddl"
+	"github.com/cloudspannerecosystem/harbourbridge/webv2/session"
+)
+
+func TestReviewTableSchemainterleaved(t *testing.T) {
```

Comment:
Add a test case for non interleaved table and also for change in datatypes in both cases.

Reply:
Non interleaved test cases are already added in review_table_schema_test.go. Will move the test case in this file to review_table_schema_test.go file to make it more consistent.